docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='_defining_a_data_model']
 = Viewing the *Mortgage Process* data objects
 
-The goal of this chapter is to introduce you to the *Mortgage Process* data objects.
+The goal of this chapter is to introduce you to the *Mortgage Process* data objects.  This tutorial utilizes the predefined data objects in the *Mortgage Process* sample project to save time in completing the mortgage application scenario. 
```

Comment:
This got a bit confusing. It is in intro to "3.2" but speaks of "this chapter" (chapter is the entire "3") and then even "this tutorial".

Reply:
Agreed and simplified. Good catch!